src/components/Hero.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import styled from 'styled-components'
+import heroImg from '../images/skyline.svg'
 
 const StyledHeroSection = styled.section`
```

Comment:
Let's nest all the other styled components you've made as regular elements in this styled component Like you did on the nav PR.